trusted-committer/de/03-keeping-the-community-healthy-de.asciidoc

```diff
@@ -0,0 +1,50 @@
+Menschen neigen auch dazu sich zu beteiligen, weil die Mitarbeit in einer _InnerSource Community_ eine hervorragende Gelegenheit bietet, neue Fähigkeiten zu erlernen und persönlich zu wachsen.
+Auch dies ist ein Aspekt, in dem die Rolle des _Trusted Committers_ wirklich wichtig ist.
+_Trusted Committers_ sind oft Mentoren für jüngere Entwickler, und verwenden ausdrücklich Zeit während der Pull Requests, um nicht nur auf Verbesserungspotentiale hinzuweisen, sondern auch im Detail zu erklären, warum und wie etwas
+verbessert werden muss. Sie liefern die Theorie oder das Hintergrundwissen zu Verbesserungen und schlagen die besten Möglichkeiten vor, Änderungen oder Verbesserungen zu implementieren.
+Dadurch können _Trusted Committers_ die Lerngeschwindigkeit ihrer Organisation weit über die traditioneller Software-Entwicklungsorganisationen erhöhen.
+
+Wir glauben, dass _Trusted Committers_ das Anwerben und Betreuen während der Pull Requests über das Erreichen von Veröffentlichungsterminen stellen sollten, außer es sprechen triftige Gründe dagegen.
+Gute Betreuung während eines Pull Requests führt zu höherem Vertrauen und Einbindung von https://innersourcecommons.org/resources/learningpath/contributor/index[_Contributors_], was im Gegenzug zu mehr Mitarbeit führt.
+Wir werden das in https://innersourcecommons.org/resources/learningpath/trusted-committer/04/["Upleveling the Community"] näher betrachten.
+
+Schlussendlich gibt es Menschen, die sich in _InnerSource Communities_ engagieren, weil sie sich dort auf das Entwickeln von Software konzentrieren können, anstatt Aufgaben nachzugehen, die häufig als überflüssig oder unnötig angesehen werden.
+Dies trifft häufig auf große Unternehmen mit einem starken Fokus auf Prozesse zu.
+Die Aufgabe des _Trusted Committers_ in diesem Zusammenhang ist es, sicherzustellen dass die https://innersourcecommons.org/resources/learningpath/contributor/index[_Contributors_] sich auf ihre Projekte konzentrieren können, indem sie
+sinnvolle Richtlinien für die Mitarbeit kommunizieren und einsetzen.
+
+Ein wichtiger Aspekt dieser Richtlinien ist, was wir in Pull Requests  _signaling_ nennen: Wie soll ein Kommentar aussehen? Was bedeutet es, wenn ich einen Kommentar "_like_" oder "_+1_" vergebe?
+Wie unterscheidet sich beim _@mentioning_ die Verwendung eines _/CC_-Vorzeichens von einem _/FYI_-Vorzeichen? Allgemein gesprochen müssen _Trusted Committers_ sicherstellen, dass der Prozess, im Projekt mitzuarbeiten, nicht mehr
+Probleme verursacht, sondern die Organisation unterstützt, Probleme im Projekt zu identifizeren und zu lösen. Schließlich sollten _Trusted Committers_ ihre Organisation dazu befähigen, prozessbezogene Probleme zu finden 
+und diese so gut wie möglich selbst anzupassen und zu verbessern.
+
+Damit _Trusted Committers_ alle diese Verantwortlichkeiten erfüllen können, ist es wichtig, dass sie regelmäßig mit den Mitgliedern der Organisation kommunizieren und ein Gespür für die allgemeine Stimmung entwickeln. 
```

Comment:
remove trailing s in _Trusted_Committers_

Reply:
done